src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.8.9
+-----
+
+* [BC BREAK] fixed inheritance of the `autowire` flag
```

Comment:
Like @jakzal, I think it's more a bug than a a BC break. Anyway, we should either remove the `[BC BREAK]` here or merge this PR on master.